ConvertAnyStringIntoKebabCaseStyle/Program.cs

```diff
@@ -1,12 +1,19 @@
 ﻿using System;
+using System.Text.RegularExpressions;
 
 namespace ConvertAnyStringIntoKebabCaseStyle
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.Write("Enter text convert to kebab-case-style: ");
+            var inputText = (Console.ReadLine());          
+            string pattern = @"[\W_]+";
+            string replacement = "-";
+            Regex regex = new Regex(pattern);
+            string result = regex.Replace(inputText, replacement);           
+            Console.WriteLine(result.ToLower().Trim('-'));          
```

Comment:
I would recommend you one of two : 
1) use just inputText.Trim(), it should delete all spaces on start and end of string. 
2) OR use variable 'replacement' in Trim like now, instead of '-' literal